compass/ocean/tests/overflow/hydro_vs_nonhydro/visualize.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib
+import matplotlib.pyplot as plt
+from netCDF4 import Dataset
+
+from compass.step import Step
+
+matplotlib.use('Agg')
+
+
+class Visualize(Step):
+    """
+    A step for visualizing a cross-section through the fluid
+    descend down the slope.
+
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='visualize')
+
+        for grid in ['nonhydro', 'hydro']:
```

Comment:
```suggestion
        for mode in ['nonhydro', 'hydro']:
```
I think it's clearer to change `grid` to `mode` here and below

Reply:
Change made.